### ISSUE TITLE
Unable to delete cluster

### ISSUE THREAD

**Issue by gogogreg** [kubernetes/kops#3342]
I am unable to delete a cluster I've created and get an AuthFailure. However other kops functions still work (e.g. kops get nodes).

I've tried creating new credentials in AWS but with no luck and the user has administrator access..

$ kops delete cluster domainname.com --yes

Got an error while querying for valid regions (verify your AWS credentials?): AuthFailure: AWS was not able to validate the provided access credentials
	status code: 401, request id: 8bb4ea20-7a84-4b24-ac23-ed7562760424

**Comment by gogogreg** [comment]
Fixed, I was running kops within a VM and the time drift was the issue

**Comment by GaudamThiyagarajan** [comment]
time drift? can you be specific

**Comment by vkahawala** [comment]
Same happened to me. 
I had two clusters; staging and dev and one VM per each cluster to trigger commands. Therefore I have to log in to the VM and then issue whichever command.

Staging cluster deletion was successful without any issue. But dev was complaining `AuthFailure: AWS was not able to validate the provided access credentials status code: 401`
After seeing this post, I checked the times in both VMs. 
Staging 4:53:00 UTC
Dev 5:03:00 UTC
Actual 4:55:00 UTC

Then I changed the dev time to staging time using `date +%T -s "04:53:00"` and the deletion successful
